ENCODE_release.py

```diff
@@ -348,6 +359,61 @@ def has_inactive_pipeline(self, file_object):
                 return [p['@id'] for p in pipelines]
         return False
 
+    def _grab_bio_id(self, field, data):
+        """
+        Return biosample_term_id of embedded object.
+        """
+        return encodedcc.get_ENCODE(data.get(field),
+                                    self.connection).get('biosample_term_id')
+
+    def _get_associated_term_id(self, name, data):
+        """
+        Find biosample_term_id associated with particular object.
+        """
+        if name not in self.hela_associated_objects:
+            raise ValueError('Can only get biosample_term_id of known '
+                             'objects:', self.hela_associated_objects)
+        if name == 'File':
```

Comment:
This if else block can is a little confusing
```
        obj_id = None
        if name == 'File':
            # Get biosample_term_id in file.dataset.
            obj_id = data.get('dataset')
        elif name == 'Replicate':
            # Get biosample_term_id in replicate.experiment.
            obj_id = data.get('experiment')
        elif name == 'Library':
            # Get biosample_term_id in library.biosample.
            obj_id = data.get('biosample')
        else:
            # For experiments and biosamples.
            biosample_term_id = data.get('biosample_term_id')
        if obj_id:
            # Return biosample_term_id of embedded object.
            biosample_term_id = encodedcc.get_ENCODE(
                obj_id,
                self.connection
            ).get('biosample_term_id')
        return biosample_term_id
```
I believe this accomplishes the same thing and doesn't need the extra function.